### PR TITLE
Service

### PR DESCRIPTION
**Pull request by T-Syme**
The service tests have not been finalized - missing the BeforeEach part, but the rest should be complete

### CODE REVIEW

**Review by JDziewonski98 — COMMENTED** [review]
looks good to me  so far

File: RepairShop-Backend/src/test/java/ca/mcgill/ecse321/repairshop/service/TestServiceService.java [hunk 1]
```diff
@@ -1,4 +1,63 @@
...
+            fail();
+        }
+
+        assertNotNull(service);
+        assertEquals(name, service.getName());
+
+    }
+
+    @Test // invalid service
+    public void testCreateServiceEmpty() {
```
Comment (sandyl289) [inline RepairShop-Backend/src/test/java/ca/mcgill/ecse321/repairshop/service/TestServiceService.java:87]: I think we'll need to add a "testCreateServiceNull" test like in the tutorial notes ( See [Tutorial's complete test suite](https://gist.githubusercontent.com/imbur/d8836dc9f44ece65186a43ac80a259fd/raw/ff59449ad4dfadfc2c54bb317649300f6ae2409c/TestEventRegistrationService.java))
image


Other than that, everything seems good

**Review by sandyl289 — APPROVED** [review]

File: RepairShop-Backend/src/test/java/ca/mcgill/ecse321/repairshop/service/TestServiceService.java (hunk 1 above)
Comment (T-Syme) [reply RepairShop-Backend/src/test/java/ca/mcgill/ecse321/repairshop/service/TestServiceService.java:87]: Done

**Comment by T-Syme** [comment]
Most changes are complete. Will open a new pull request to merge to main

**Pull request closed without merging**